src/venue-service/models/index.ts

```diff
@@ -3,3 +3,5 @@ export { SeatAttributes } from './seat-attributes';
 export { VenueDetails } from './venue-details';
 export { SeatSettings } from './seat-settings';
 export { ChartInfo } from './chart-info';
+export { Facility } from './facility';
+export * from './terminal';
```

Comment:
I think on this level we should either specify what we want to export or export everything from all modules. Might be helpful if you would describe all types because it will be easier to read in the IDE.